tests/subscriptions/test_codecs.py

```diff
@@ -164,7 +164,7 @@ def test_subscription_task_result_encoder() -> None:
         task_result.task.task.subscription.identifier
     )
     assert payload["request"] == request.original_body
-    assert payload["result"] == result
+    assert payload["result"]["data"] == result["data"]
```

Comment:
Why not remove the `profile` and `trace_output` from the expected value?

Reply:
Probably could've but I don't think it makes too much difference since only this one actually gets used